jota/src/test/java/org/iota/jota/IotaAPITest.java

```diff
@@ -192,15 +255,15 @@ public void shouldPrepareTransfer() throws ArgumentException {
     @Test
     @Tag("IntegrationTest")
     public void shouldPrepareTransferWithInputs() throws ArgumentException {
-        List<Input> inputlist = new ArrayList<>();
+        List<Input> inputList;
         List<Transfer> transfers = new ArrayList<>();
 
         GetBalancesAndFormatResponse rsp = iotaAPI.getInputs(TEST_SEED1, 2, 0, 10, 0);
-        
-        inputlist.addAll(rsp.getInputs());
+
+        inputList  = new ArrayList<>(rsp.getInputs());
```

Comment:
nice :) Its just a test so optimization is not that important, but i like it!

Reply:
Yes, after the push I saw it on several other places. I think we can re-organise the tests in an other PR.